docs/includes/apiargs-MongoDBCollection-method-aggregate-option.yaml

```diff
@@ -35,6 +35,18 @@ post: |
   version of MongoDB, this option will be ignored.
 ---
 arg_name: option
+name: comment
+type: string
+description: |
+  Users can specify an arbitrary string to help trace the operation through the
+  database profiler, currentOp, and logs.
+
+  .. versionadded:: 1.3
```

Comment:
Good catch! I just came across #423 while looking through closed PRs and was going to pop in to ask about this, but you beat me to it :+1: